data/categories.txt

```diff
@@ -12046,7 +12046,7 @@ cs:4Parkovací automat
 da:4Parkeringsbilletmaskine
 de:4Parkautomat|Parkscheinautomat|Parkticketautomat
 el:Εισιτήρια στάθμευσης
-es:Máquina de pago de tique de aparcamiento
+es:Máquina de pago de tique de aparcamiento|Máquina de pago de tique de parking|Máquina de pago de tique de estacionamiento|Parquímetro
```

Comment:
Do you really think that someone will type `Máquina de pago de tique de estacionamiento` in the search?... I would suggest adding a number of characters before the first keyword to show a hint when the user starts typing instead of adding unlikely complex synonyms. Like that:
`4Máquina de pago de tique de aparcamiento`

Reply:
Maybe not the full string but it’s quite likely that someone types "tique estacionamiento" and that works with these strings

Reply:
Would it be better to add "tique estacionamiento" instead?

Reply:
Is there a problem in having these long strings? These strings cover a lot of possible substrings that users could search: "pago tique", "pago aparcamiento", "máquina tique", "pago estacionamiento" and they all work without typing the full string

Reply:
Added number of characters for hints

Reply:
I've simplified "Máquina de pago de tique de aparcamiento" to "Tique de aparcamiento" but I've kept it in alternative strings in categories because I think it's useful keeping the full name for substring searching

Reply:
> Tique de aparcamiento

Althought the spanish dictionary documents it as "tique" I would also have added the more common one "[ticket](https://es.wiktionary.org/wiki/ticket)". But this is a small thing, few people are expected to search for this POI type.

Reply:
> Is there a problem in having these long strings? These strings cover a lot of possible substrings that users could search: "pago tique", "pago aparcamiento", "máquina tique", "pago estacionamiento" and they all work without typing the full string

@vng wdyt?

Reply:
> > Tique de aparcamiento
> 
> Althought the spanish dictionary documents it as "tique" I would also have added the more common one "[ticket](https://es.wiktionary.org/wiki/ticket)". But this is a small thing, few people are expected to search for this POI type.

Added. Thanks!